packages/kokkos-kernels/src/common/KokkosKernels_Handle.hpp

```diff
@@ -331,6 +337,22 @@ class KokkosKernelsHandle{
   }
 
 
+  SPADDHandleType *get_spadd_handle(){
+    return this->spaddHandle;
+  }
+
+  void create_spadd_handle(bool input_sorted) {
```

Comment:
I thought this method is idempotent. Instead it deletes old handles which means the user has the responsibilty to call it right. It is ok for this commit, as it follows spgemm style, but can you file an issue to remove this delete()  or to describe the case for it clearly.

Reply:
This is the way I followed overall. We can talk on the meeting and I can explanation more clearly

Reply:
understood. Brain doesn't have to be blocked by this.